Assignment1_q2_Singly_Linked_List.py

```diff
@@ -0,0 +1,57 @@
+from Assignment1_q2_Node import *
```

Comment:
As said above, you want to probably put the `Node` class in this file, but (generally) avoid importing `*` into a module: this means that if someone defines new things in the other module, all of them will be available (and might affect how your code works). Instead, import only what you need.